app/mixins/fetch_package_detail.js

```diff
@@ -0,0 +1,38 @@
+import Ember from "ember";
```

Comment:
Similar to my previous comments. The usage of components and mixins are a bit misunderstood.
Given the time constraints we can move forward with this.
But keep it in mind next time you have an opportunity to refactor, feel free to ping me if you have questions :)